about.html

```diff
@@ -0,0 +1,87 @@
+    <nav class="navbar nav1 navbar-expand-lg fixed-top nav_container d-inline-flex justify-content-end pb-0 pt-0"
+      role="navigation">
+      <ul class="">
+        <li><a href="#" class="nav-link1"><img src="assets/images/104498_facebook_icon.svg" alt="facebook icon"
+              class="nav-icons"></a></li>
+        <li><a href="#" class="nav-link1"><img src="assets/images/104501_twitter_icon.svg" alt="twitter icon"
+              class="nav-icons"></a></li>
+        <li><a href="#" class="nav-link1">English</a></li>
+        <li><a href="#" class="nav-link1">My Page</a></li>
+        <li><a href="#" class="nav-link1">Logouts</a></li>
+      </ul>
+    </nav>
+    <a class="menu justify-content-start"><img src="assets/images/menu_button.svg" alt="menu button"></a>
+
+    <nav class="navbar nav2 navbar-expand-lg fixed-top nav_container d-inline-flex justify-content-end"
+      role="navigation">
+      <ul class="">
+        <li id="small-logo"><a href="index.html"><img class="logo-small" src="assets/images/BTR-Logo.png" alt="btr logo"></a></li>
+        <li><a href="about.html" class="nav-link2">About</a></li>
+        <li><a href="#program" class="nav-link2">Program</a></li>
+        <li><a href="#" class="nav-link2">Join</a></li>
+        <li><a href="#" class="nav-link2">Sponsors</a></li>
+        <li><a href="index.html" class="nav-link2 logo">BTR Concert</a></li>
+      </ul>
+    </nav>
+    <div class="menu_opt">
+      <input type="button" name="close" value="X" class="close_menu" id="exit">
+      <ul class="menu_list">
+        <li class="menu_items"><a href="index.html">BTR Concert</a></li>
+        <li class="menu_items"><a href="about.html">About</a></li>
+        <li class="menu_items"><a href="#program">Program</a></li>
+        <li class="menu_items"><a href="#">Join</a></li>
+        <li class="menu_items"><a href="#">Sponsors</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <section class="about" id="about-page">
+    <div class="page-about">
+      <div class="overlay">
+        <div class="home-text about-text">
+          <h3>"It's about time to connect with your roots."</h3><br />
+          <h1>BACK TO ROOTS<br /> GLOBAL CONCERT 2012</h1>
+        </div>
+        <div class="home-p">
+          <p>A joyful celebration believing in the value of connecting with one's roots through musical entertainment.
+            Become the part of this year's concert by buying your ticket online.</p>
+        </div>
+        <div class="about-p2">
+          <p>Please contact us per Email for any further questions about the BTR Global Concert 2012!</p>
+          <p>info@brtgc.org</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+    <footer class="foot1 pt-5">
+      <div class="btr-div"><img class="btr" src="assets/images/BTR-Logo.png" alt="BTR Logo"></div>
+      <div class="foot1-div">
+        <p class="foot1-p">2012 Back To Roots. Some rights Reserved.</p>
+      </div>
+    </footer>
```

Comment:
[OPTIONAL] The previous reviewer also missed this. In the footer of the about page for the mobile screen size only the background colour is supposed to be white according to Cindy's design.

|Template|
|--------|
|![image](https://user-images.githubusercontent.com/62938735/192121203-e186e8dc-edd9-4cce-8bd9-e6f4c3cc0219.png)|